### ISSUE TITLE
Does not have the standard 'puts' alias 'p' 

### ISSUE THREAD

**Issue by wade-welles** [grubby/grubby#7]

**Comment by tjarratt** [comment]
True. I never got around to that. Are you interested in developing this, Wade ?

I haven’t touched grubby in a few years but every so often I get the itch to dig back in. It’s very much deprecated but imho a fun project to hack on. Iirc the ruby version I was trying to target was something like 2.1 or maybe 2.0 or maybe even 1.9 for simplicity. It’s fallen fairly far behind the other rubies out there. Lots of catching up to do. 

Sent From A Very Small Keyboard

**Comment by tjarratt** [comment]
Huh well today I learned that `Kernel#p` is different from `Kernel.puts`, in that the former calls `#.inspect` on the arguments, and that it is variadic, accepting of zero or more arguments[1].

Thanks for sending me down this rabbithole @wade-welles. 

[1]https://ruby-doc.org/core-2.6.5/Kernel.html#method-i-puts
